README.md

```diff
@@ -81,6 +81,8 @@ Creates a new GoodInflux object where:
   - `[prefixDelimiter]` - Used to delimit measurement prefix arrays defined in *prefix* above. Defaults to `/`.
   - `[customLogFormatters]` - This options allows you to report specific log events based on tag (Note the first matching tag in `event.tags` will be used, it's better to have specific tags in practice). Ordinarily, log events are sent as a string. However, you may want to extract a custom data field and send it to Influx in a more searchable format. For example:
   - `[fieldTags]` - This option takes an array of strings. When logging data it will look for those fields in the data values processed and then add them to the `tags` if they are.
+  - `[measurementOverride]` - This is a function which can be provided if you would like to override which measurement to use based on other data than the event name. The function is called with the whole `event` object. Changing the measurement does _not_ affect the processing of its data. If this is provided, `prefix` and `prefixDelimiter` are both ignored.
```

Comment:
When you say "based on the data than the event name", so you mean tags, fields, time, etc? Might be good to add an example.

It sounds like adding a function like this would mean that every event needs to be processed via such a function. Any performance considerations there? Even though Hapi Good functions run off of the main event loop, they still do tax the core Node process, and do show up in Hapi profiling. Worth considering.

Reply:
The performance impact of this would be n*m where n is the complexity of your function and m is the number of logs. Realistically most use cases for this would be O(1), so performance impact should be pretty negligible.